laceworksdk/api/base_endpoint.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BaseEndpoint(object):
+    """
+    Lacework BaseEndpoint Class.
+    """
+
+    def __init__(self,
+                 session,
+                 endpoint_path,
+                 object_type):
+        """
+        :param session: An instance of the HttpSession class.
+        :param endpoint_path: The URL endpoint to use for the object type.
+        :param object_type: The Lacework object type to use for validation. (TODO)
+        """
+
+        super().__init__()
+        self._session = session
+        self._endpoint_path = endpoint_path
+        self._object_type = object_type
+
+    def build_dict_from_items(self, *dicts, **items):
+        """
+        A method to build a dictionary based on inputs, pruning items that are None
```

Comment:
this also favors the last key value... might want to indicate that as well.. that is if I pass in a dict with

```
a = {'a': 234}
b = {'a': 124} 
```

This function would return a dict with 

```
{'a': 124}
```
that is favour the last seen value of any given key.... 

unless you don't want that behaviour and you prefer to raise a KeyError or something like that if you see the same key twice?